packages/expo-screen-capture/src/ScreenCapture.ts

```diff
@@ -108,9 +110,11 @@ export function addScreenshotListener(listener: () => void): EventSubscription {
 // @needsAudit
 /**
  * Removes the subscription you provide, so that you are no longer listening for screenshots.
+ * If you prefer, you can also call `remove()` on that `Subscription` object.
```

Comment:
```suggestion
 * You can also call `remove()` on that `Subscription` object.
```

I'd suggest keeping this short (I understand this is unrelated to the contents of this PR -- happy to follow up in another one too).